testsuite/features/secondary/srv_content_lifecycle.feature

```diff
@@ -37,22 +37,23 @@ Feature: Content lifecycle
     And I follow "clp_name"
     And I click on "Attach/Detach Sources"
     And I select "SLES12-SP5-Pool for x86_64" from "selectedBaseChannel"
+    And I wait until I see "SLES12-SP5-Pool for x86_64" text
```

Comment:
If we do:
`first(:xpath, ".//*[contains(text(), 'SLES12-SP5-Pool for x86_64')]").click`

In theory, the Capybara `first` method it will wait the default timeout until it found a first web element containing this text.
So, in theory, it will not be a reason to wait again until we see the text. Unless... selecting the option, the text disappear and appear again after few milliseconds.

Reply:
Yes, but if you get my point, depending the speed of the web, the select before is already looking for that exact text, so if we select it and we wait for the same exact text, depending the speed of the web, it might just go ahead and pass it.